judge/models/contest.py

```diff
@@ -84,6 +84,7 @@ class Contest(models.Model):
     organizations = models.ManyToManyField(Organization, blank=True, verbose_name=_('organizations'),
                                            help_text=_('If private, only these organizations may see the contest'))
     og_image = models.CharField(verbose_name=_('OpenGraph image'), default='', max_length=150, blank=True)
+    logo_override = models.CharField(verbose_name=_('Logo Override'), default='', max_length=150, blank=True)
```

Comment:
"Override" should be lowercase. Also missing help text. A better name would be `logo_override_image`.